Documentation/CMakeLists.txt

```diff
@@ -52,6 +48,8 @@ add_custom_target(doc ALL
     COMMENT "Generating API documentation with Doxygen"
     VERBATIM)
 
+set_target_properties(doc PROPERTIES VERSION ${OSMSCOUT_LIBRARY_VERSION} SOVERSION ${OSMSCOUT_LIBRARY_VERSION} FOLDER "Documentation")
```

Comment:
Does setting `VERSION` and `SOVERSION` on a custom target do anything useful? Same question for executable targets.

Reply:
Useful for package generation.